### PR TITLE
Vp test

### PR DESCRIPTION
**Pull request by eugenegodun**

### CODE REVIEW

File: wdio.conf.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ exports.config = {
     // backend you should define the host address, port, and path here.
     //
     host: '0.0.0.0',
-    port: 4444,
+    port: 4445,
```
Comment (IrinaKolodiazhnaya) [inline wdio.conf.js]: @eugenegodun why did you change port to another?

File: test/textChat/testSendMessage.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            await loginPage.loginWithCreds(users.stageStudentCreds.usernameStudent,
+                users.stageStudentCreds.passwordStudent);
+            await homePage.clickLaunchPreplySpaceLink();
+            // await checkoutPage.clickCheckoutBtnSavedCard();
+            await preplySpacePage.waitForPrepplySpacePageAppeared();
+            let btn = await preplySpacePage.getSendMessageButton();
+            expect(await btn).to.include("Send message");
+        });
+    });
+});
```
Comment (IrinaKolodiazhnaya) [inline test/textChat/testSendMessage.js]: @eugenegodun Why did you add this test to the folder test/textChat ?
we are using this one webdriverio/test for converting via babel

Comment (vadymRudenkoPreply) [reply test/textChat/testSendMessage.js]: +1, @eugenegodun we will delete folders ./pageObject, ./test, ./WebActions, ./driverHelper - this is folders we used before webdriverio, we left them to compare it with webdriverio solution. So if your solution works fine using wdio you may delete this

File: webdriverio/po/preplySpace.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import Page from './page';
+
+class PreplySapcePage extends Page {
+
+    get btnSendMessage() { return browser.element("//button[@type='submit']") };
+    get textarea() { return browser.element("//div[@class='add-message__field']//textarea"); }
+    get launchPreplySapceLink() { return browser.element("//a[contains(@href, 'chat/room')]") };
+    get chat() { return browser.element("//div[@class='chat']") };
+    get lastMessage() { return browser.element(".chat__messages .message:last-child .Linkify") };
+
```
Comment (vadymRudenkoPreply) [inline webdriverio/po/preplySpace.js]: we talked about adding automation attribute, is it possible to do this in this test?

File: test/textChat/testSendMessage.js (hunk 2 above)
Comment (eugenegodun) [reply test/textChat/testSendMessage.js]: I've started this task before webdriverio :) Will delete it.

File: wdio.conf.js (hunk 1 above)
Comment (eugenegodun) [reply wdio.conf.js]: will return back. Just didn't work for me on this post.

File: webdriverio/po/preplySpace.js (hunk 3 above)
Comment (eugenegodun) [reply webdriverio/po/preplySpace.js]: we can, but need to merge this repo to apollo

File: webdriverio/test/testSendMessage.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+describe('Preply Space tests', () => {
+    describe('Chat', () => {
+        it('send message', () => {
+            const defaultTimeout = 10000; // Not sure about this pattern
+            HomePage.open();
+            HomePage.loginBtn.click();
+            LoginPage.usernameField.setValue(Users.student.username);
+            LoginPage.passwordField.setValue(Users.student.password);
+            LoginPage.submitLoginBtn.click();
+            PreplySpacePage.launchPreplySapceLink.click().pause(2000);
```
[the remark this answers is not in the document]
Comment (eugenegodun) [reply webdriverio/test/testSendMessage.js]: will try
